partition.cpp

```diff
@@ -1237,7 +1244,8 @@ void TWPartition::Setup_Data_Media() {
 			Make_Dir("/sdcard", false);
 			Symlink_Mount_Point = "/sdcard";
 		}
-		if (Mount(false) && TWFunc::Path_Exists(Mount_Point + "/media/0")) {
```

Comment:
Here I am having doubt

Reply:
https://gerrit.twrp.me/c/android_bootable_recovery/+/5041/22/partition.cpp#1204

Need to add

Reply:
Working fine for all devices?

Reply:
> Working fine for all devices?

Tested on RMX1941 format data & internal shows after format but format data is still broken on some device

Reply:
one more last thing, is it missing authorship?